apps/golden-sample-app/src/app/app.module.ts

```diff
@@ -80,7 +80,12 @@ import { IdentityAuthModule } from '@backbase/identity-auth';
     {
       provide: APP_INITIALIZER,
       multi: true,
-      deps: [OAuthService, CookieService, AuthEventsHandlerService],
+      deps: [
+        OAuthService,
+        AuthService,
+        CookieService,
+        AuthEventsHandlerService,
```

Comment:
this doesn't look right, it doesn't matches the parameters of the factory

```suggestion
        AuthService,
        CookieService
```